src/themes/device.js

```diff
@@ -1,4 +1,5 @@
 export const size = {
+    mobileXS: 280,
```

Comment:
Why do we need to support all the way down to 280px here?

Reply:
removed